build/azure-pipelines.release-template.yml

```diff
@@ -164,13 +167,23 @@ stages:
           - script: go run mage.go UseXBuildBinaries
             displayName: "Setup Bin"
           - task: Docker@1
-            displayName: Docker Login
+            displayName: Docker Hub Login
             inputs:
               containerRegistryType: Container Registry
-              dockerRegistryEndpoint: ${{parameters.registry}}
+              dockerRegistryEndpoint: ${{parameters.registry}} # Log in with the saved credentials for the destination registry
               command: login
           - script: REGISTRY=${{parameters.registry}} make publish-images
-            displayName: "Publish Docker Images"
+            displayName: "Publish Docker Images to ${{parameters.registry}}"
+          - task: Docker@1 # Support publishing to our old registry on Docker Hub and to our new location on GHCR while we migrate
+            displayName: GHCR Login
+            condition: ${{parameters.dualPublish}}
+            inputs:
+              containerRegistryType: Container Registry
+              dockerRegistryEndpoint: ghcr.io/getporter
+              command: login
+          - script: REGISTRY=ghcr.io/getporter make publish-images
```

Comment:
Question here.  This line should either omit this `REGISTRY` value _or_ be set to `REGISTRY=${{parameters.registry}}`, right?  Otherwise, if I'm reading the make target correctly, it will publish to `ghcr.io/getporter` twice.

Reply:
What I am trying to do is always publish to parameters.registry (which is currently set to docker.io/getporter). Then when dual publish is set, publish to ghcr.io/getporter as well.

So the pairs of tasks are 

* login to parameters.registry (which is either getporter or getporterci)
* push to parameters.registry
* login to ghcr.io/getporter
* push to ghcr.io/getporter

Looks like azure devops supports for loops now, so an alternate way we could try it is to define parameters.registry as an array, https://stackoverflow.com/a/62871342